README.md

```diff
@@ -143,23 +143,29 @@ See the [changelog](CHANGELOG.md) for details about specific releases.
 <br><hr>
 <h3>Development Process</h3>
 
+CUB and thrust depends on each other. To build CUB, you need to clone the
+same version of thrust.
+
 CUB uses the [CMake build system](https://cmake.org/) to build unit tests,
 examples, and header tests. To build CUB as a developer, the following
 recipe should be followed:
 
 ```
 # Clone CUB repo from github:
 git clone https://github.com/NVIDIA/cub.git
+# Clone the same version of thrust:
+git clone --recursive https://github.com/NVIDIA/thrust.git /path/to/thrust_repo
```

Comment:
Does it look good right now? Seems that build thrust will not automatically build cub, and thrust's tutorial does not mention how to build cub.

Reply:
> Does it look good right now?

I was thinking of just linking to `https://github.com/NVIDIA/thrust/blob/main/CONTRIBUTING.md` directly from this document and pointing out the relevant CUB CMake options.

It might be easier if I write this section since it'll require a lot of familiarity with our build system. Would you mind if I pushed some commits to this PR?

> Seems that build thrust will not automatically build cub, and thrust's tutorial does not mention how to build cub.

It's documented in [this section](https://github.com/NVIDIA/thrust/blob/main/CONTRIBUTING.md#cuda-specific-cmake-options) of the Thrust docs. Setting the CMake option `THRUST_INCLUDE_CUB_CMAKE=ON` will enable CUB's tests, examples, etc.

Reply:
Yeah, feel free to push here.